pkg/apis/triggers/v1alpha1/interceptor_types.go

```diff
@@ -45,7 +46,7 @@ type InterceptorResponse struct {
 	// See TEP-0022. Naming TBD.
 	Extensions map[string]interface{} `json:"extensions,omitempty"`
 	// Continue indicates if the EventListener should continue processing the Trigger or not
-	Continue bool `json:"continue,omitempty"`
+	Continue bool `json:"continue"` // Adding omitempty will remove the continue field when the value is false.
```

Comment:
nit: Reworded to make it clear to not do this (on first pass I read this like a TODO 😅 )

```suggestion
	Continue bool `json:"continue"` // Don't add omitempty - this will remove the continue field when the value is false.
```

Reply:
done.